schema-derivatives-options.graphql

```diff
@@ -1245,9 +1278,17 @@ type ActiveAccount @entity {
   id: Bytes!
 }
 
+enum OptionType {
```

Comment:
let's place this at the beginning where all the other enums are

Reply:
Changed at 5b75305b341ea53c57e08fee4dab7eb7420db7e9